pom.xml

```diff
@@ -119,6 +119,18 @@
       </plugins>
     </pluginManagement>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.6.1</version>
```

Comment:
Do we need to declare version here, why not inherit from parent?

Reply:
I've just noticed that Evolution warns about it. Will remove.